index.md

```diff
@@ -83,37 +62,58 @@ We weight grades as follows:
 | Final Project | 40% | 25% |  
 | 4th Credit Project | | 15% |  
 
-You can expect the cutoffs to be approximately as follows
-98 - 100: A+  
-93 - 98: A
-90 - 93: A-
-87 - 90: B+
-83 - 87: B
-80 - 83: B-
-77 - 80: C+
-73 - 77: C
-70 - 73: C-
-60 - 70: D
-Below 60: F
+You can expect the cutoffs to be approximately as follows: 
 
+- [90, 100]: A-, A, A+
+- [80, 90): B-, B, B+
+- [70, 80): C-, C, C+
+- [60, 70): D-, D, D+
+- [0, 60): F
 
+## Assignments ##
+We expect to have 4 assignments over the course of the semester. In each of these, students will work in pairs to solve a specific implementation problem in the virtual reality lab in 4107 Siebel Center.
 
-## Textbooks ##
-Main text, required: [Steven M. LaValle, Virtual Reality, 2016](http://vr.cs.uiuc.edu).
+- Read the submission instructions in each handout carefully!
+- You must work in pairs on all assignments. We recommend using Piazza to find a group.
+- We will accept assignments only up to two days late, with a 20% penalty.
 
-Optional: George Mather, Foundations of Sensation and Perception: Psychology Press; 2 edition, 2009.
+See the [Assignments page](/assignments) for more details.
 
-Optional: Peter Shirley, Michael Ashikhmin, and Steve Marschner, Fundamentals of Computer Graphics, A K Peters/CRC Press; 3 edition, 2009.
+## Exams ##
+This class will have three midterms and no final exam. Each exam is 50 minutes long. You will need to make a reservation in the [CBTF](https://cbtf.engr.illinois.edu) to take the exam.
 
-See the [Extra Material](/extra-material) page for other, recommended readings.
+- Midterm 1: Feb. 10 - Feb. 12
+- Optional second chance exam for Midterm 1: Feb. 17 - Feb. 19
+- Midterm 2: Mar. 10 - Mar. 12
+- Optional second chance exam for Midterm 2: Mar. 25 Mar. 26
+- Midterm 3:  Apr. 28 - Apr. 30
 
-## [Piazza](https://piazza.com/illinois/spring2019/cs498vr) ##
-We will use Piazza for answering questions and administering the course.
+All exams are closed-everything (no book, notes, calculator, etc.). See the [CBTF policies](https://cbtf.engr.illinois.edu/for-students/policies.html) for more detail. You are responsible for all material covered in lectures and assignments.For DRES accommodations see [CBTF DRES Students](https://cbtf.engr.illinois.edu/for-students/dres-students.html)
```

Comment:
I believe the CBTF provides calculators now

Reply:
They provide basic caclulators. I suspect they wouldn't help much here. There is a missing space between "assignments." and "For".

Reply:
Edited to include calculators